rocky/rocky/views/mixins.py

```diff
@@ -97,33 +97,44 @@ def get_origins(
         reference: Reference,
         organization: Organization,
     ) -> tuple[list[OriginData], list[OriginData], list[OriginData]]:
+        results = [], [], []  # type: tuple[list[OriginData], list[OriginData], list[OriginData]]
```

Comment:
```suggestion
        results: tuple[list[OriginData], list[OriginData], list[OriginData]] = [], [], []
```

But imo it would be better to separate them as variables and then combine them as tuple at the end (either as a new variable, or simply `return x, y, z`. This would make it more readable (since it's easier to read variables than tuple indices) and slightly faster by avoiding tuple indexing

Reply:
I was thinking about returning a dictionary instead of this list, as its more descriptive. But did not want to grow the PR too much.